composer.json

```diff
@@ -18,27 +18,29 @@
     "config": {
         "sort-packages": true,
         "platform": {
-            "php": "7.3.99"
+            "php": "7.4.99"
+        },
+        "allow-plugins": {
+            "dealerdirect/phpcodesniffer-composer-installer": true
         }
     },
     "extra": {
     },
     "require": {
-        "php": "^7.3 || ~8.0.0 || ~8.1.0",
-        "container-interop/container-interop": "^1.2",
-        "laminas/laminas-eventmanager": "^3.4",
-        "laminas/laminas-http": "^2.15",
+        "php": "^7.4 || ~8.0.0 || ~8.1.0",
+        "laminas/laminas-eventmanager": "^3.5",
+        "laminas/laminas-http": "^2.16",
         "laminas/laminas-modulemanager": "^2.8",
-        "laminas/laminas-router": "^3.5",
-        "laminas/laminas-servicemanager": "^3.7",
-        "laminas/laminas-stdlib": "^3.6",
-        "laminas/laminas-view": "^2.14"
+        "laminas/laminas-router": "^3.9",
+        "laminas/laminas-servicemanager": "^3.16.0",
+        "laminas/laminas-stdlib": "^3.13",
+        "laminas/laminas-view": "^2.23"
     },
     "require-dev": {
         "http-interop/http-middleware": "^0.4.1",
         "laminas/laminas-coding-standard": "^1.0.0",
         "laminas/laminas-json": "^3.3",
-        "laminas/laminas-psr7bridge": "^1.0",
+        "laminas/laminas-psr7bridge": "^1.7.0",
         "laminas/laminas-stratigility": ">=2.0.1 <2.2",
```

Comment:
Just noticed that stratigility `^2` is now ~2y old.

@Xerkus can you provide some guidance on how to get out of this situation? This constraint should be either removed or bumped, but @gsteel said it can't be done without BC implications.